### PR TITLE
docs: Mention the importance of synchronized clocks for remote worker machines

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/41408

### CODE REVIEW

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -326,9 +326,11 @@ start` instead.
 
 == Run workers
 
-Workers are processes running virtual machines to perform the actual
-testing. They are distributed as a separate package and can be installed on
-multiple machines but still using only one WebUI.
+openQA workers are services running backends to perform the actual testing.
+The testing is commonly performed by running virtual machines but depending on
+the specific backend configuration multiple different options exist. openQA
```
Comment (Martchus) [inline docs/Installing.asciidoc:331]: I would use only "multiple" xor "different".

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -326,9 +326,11 @@ start` instead.
...
+workers are distributed as a separate package and can be installed on multiple
+machines but still using only one WebUI.
```
Comment (Martchus) [inline docs/Installing.asciidoc:333]: I suppose we usually spell it "web UI" in the documentation.

File: docs/Installing.asciidoc [hunk 3]
```diff
@@ -326,9 +326,11 @@ start` instead.
 
 == Run workers
 
-Workers are processes running virtual machines to perform the actual
-testing. They are distributed as a separate package and can be installed on
-multiple machines but still using only one WebUI.
+openQA workers are services running backends to perform the actual testing.
```
Comment (Martchus) [inline docs/Installing.asciidoc:329]: I would rephrase the caption to "Run openQA workers" and remove the "openQA" in the paragraph. Due to the heading it should be clear enough what kind of workers the paragraph is about. This way we don't have to start the sentences weirdly with a small letter.

File: docs/Installing.asciidoc [hunk 4]
```diff
@@ -380,6 +382,10 @@ installed 'os-autoinst' from packages make sure to pass `--isotovideo` option
 to point to the checkout dir where isotovideo is, not to `/usr/lib`! Otherwise
 it will have trouble finding its perl modules.
 
+If you start openQA workers on a different machine than the WebUI host make
```
Comment (Martchus) [inline docs/Installing.asciidoc:385]: I suppose we usually spell it "web UI" in the documentation.

File: docs/Installing.asciidoc (hunk 3 above)
Comment (okurz) [reply docs/Installing.asciidoc]: ok, fine. I have followed that advice but now I changed the reference to the package as I think it can be confusing to read that "they can be installed" when actually the (single) package is installed. Please check again the updated text.
